obs2ioda-v2/src/netcdf_cxx_mod.f90

```diff
@@ -328,7 +333,7 @@ function netcdfSetFill(netcdfID, varName, fillMode, fillValue, groupName)
         end select
     end function netcdfSetFill
 
-    ! netcdfPutAtt:
+    ! netcdfPutAttScalar:
```

Comment:
It looks like `netcdfPutAttScalar` needs to be changed to `netcdfPutAtt` on line 336.